stable/toxiproxy/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
Let's delete this file, I don't think we need tests as it's covered in the CI of this repo.